.devcontainer/devcontainer.json

```diff
@@ -29,6 +29,7 @@
   // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
   // "remoteUser": "root",
 
+  "containerUser": "vscode",
```

Comment:
Looks like the line in https://github.com/rails/devcontainer/blob/3d1b0cf63f43a1818f4efbb49c58ae7d78ddf4ac/images/ruby/.devcontainer/devcontainer.json#L23 has no effect. 

See also https://github.com/devcontainers/cli/issues/708.